lambda_function.py

```diff
@@ -21,9 +21,12 @@
 collection_table_name = os.getenv('DYNO_Collection_TABLE')
 archive_table_name = os.getenv('DYNO_Archive_TABLE')
 app_img_root_path = os.getenv('APP_IMG_ROOT_PATH')
-noid_template = os.getenv('NOID_Template')
```

Comment:
If you don't need this anymore, please remove https://github.com/vt-digital-libraries-platform/S3toDDB/pull/6/files#diff-363d481fe07d1094db2338998d381b71R26-R28 and https://github.com/vt-digital-libraries-platform/S3toDDB/pull/6/files#diff-0730bb7c2e8f9ea2438b52e419dd86c9R25

Reply:
Removed NOIDTemplate from template.yaml and readme.md